src/app/example-module/another-component/another-component.component.ts

```diff
@@ -1,4 +1,7 @@
 import { Component, OnInit } from '@angular/core';
```

Comment:
W `example-module` generalnie formater robi rzeczy w różnych miejscach

Reply:
no nie mamy formatera do JS usalonego
a domyślny bojkotuję

Reply:
Doskonała okazja na ustalenie formatera do jsa

Reply:
bosz, nie, tylko nie js
Mam pomysł

Reply:
albo nie mam

Reply:
W kwestii formatera - jest sobie na przykład https://prettier.io/. Co prawda do IntelliJ wymaga dodatkowego pluginu (https://plugins.jetbrains.com/plugin/10456-prettier), acz zdaje się robić robotę lepiej niż domyślny. ;)

Reply:
Spoks, mi nie chodzi o to, że to jest najpiękniejsze, tylko o to, żeby wszyscy korzystali z tego samego, zamiast sobie potem zmieniać nawzajem

Reply:
A, okej. To mamy przecież xmla w `.idea/codeStyles` - jak sądzę powinien od zapewniać jednolitość formatowania kodu u wszystkich.

Reply:
W sumie obojętne, na co się zgodzicie/zgodzimy (a jeśli jest to IntelliJ, to również obojętne, co konkretnie będzie w xmlu), tylko żeby była jednoznaczna spójna strategia :)

Reply:
To prosiłbym kogoś o sformatowanie tego i wysłanie, bo mój intellij ewidentnie się uparł

Reply:
Apdejtowałeś IntelliJa może? Bo u mnie to 2019.1.1 robi dziwne rzeczy. Pushnąć Ci tutaj po prostu?

Reply:
Tak, mam tego (chyba) najnowszego.
Dzięki!